app/server/api/ussd_api.py

```diff
@@ -23,6 +25,7 @@ class ProcessKenyaUssd(MethodView):
         Method returns a text response to be displayed to the user's screen
         All responses mid-session begin with CON. All responses that terminate the session begin with the word END
     """
+    @show_all
```

Comment:
oops i should have left all my new changes as a commit so you can see them. instead i'll just comment here what changed. made this `show_all`, made what's stored on session full `user_input`